example/spec/parallel/testAPI/bulkOperationsSuccessSpec.js

```diff
@@ -0,0 +1,293 @@
+        beforeAllSucceeded = true;
+      } catch (error) {
+        console.log(error);
+      }
+    });
+
+    afterAll(async () => {
+      // Must delete rules before deleting associated collection and provider
+      await deleteRule({ prefix, ruleName: get(ingestGranuleRule, 'name') });
+
+      await pAll(
+        [
+          () => deleteProvider({ prefix, providerId: get(provider, 'id') }),
+          () => deleteCollection({
+            prefix,
+            collectionName: get(collection, 'name'),
+            collectionVersion: get(collection, 'version'),
+          }),
+          () => granules.deleteGranule({ prefix, granuleId }),
+        ],
+        { stopOnError: false }
+      ).catch(console.error);
+    });
+
+    it('ingested granule is archived', () => {
+      expect(beforeAllSucceeded).toBeTrue();
+      expect(ingestedGranule).toBeTruthy();
+    });
+
+    it('returns a status code of 202', () => {
+      expect(beforeAllSucceeded).toBeTrue();
+      expect(postBulkGranulesResponse.statusCode).toEqual(202);
+    });
+
+    it('returns an Async Operation Id', () => {
+      expect(beforeAllSucceeded).toBeTrue();
+      expect(postBulkOperationsBody.id).toMatch(/[\da-f]{8}(?:-[\da-f]{4}){3}-[\da-f]{12}/);
+    });
+
+    it('creates an AsyncOperation', async () => {
+      expect(beforeAllSucceeded).toBeTrue();
+
+      const getAsyncOperationResponse = await apiTestUtils.getAsyncOperation({
+        prefix,
+        id: postBulkOperationsBody.id,
+      });
+
+      expect(getAsyncOperationResponse.statusCode).toEqual(200);
+
+      const getAsyncOperationBody = JSON.parse(getAsyncOperationResponse.body);
+
+      expect(getAsyncOperationBody.id).toEqual(postBulkOperationsBody.id);
+    });
+
+    it('runs an ECS task', async () => {
+      expect(beforeAllSucceeded).toBeTrue();
+
+      // Verify that the task ARN exists in that cluster
+      const describeTasksResponse = await ecs().describeTasks({
+        cluster: clusterArn,
+        tasks: [taskArn],
+      }).promise();
+
+      expect(describeTasksResponse.tasks.length).toEqual(1);
+    });
+
+    it('eventually generates the correct output', async () => {
+      expect(beforeAllSucceeded).toBeTrue();
+
+      await ecs().waitFor(
+        'tasksStopped',
+        {
+          cluster: clusterArn,
+          tasks: [taskArn],
+        }
+      ).promise();
+
+      const getAsyncOperationResponse = await apiTestUtils.getAsyncOperation({
+        prefix,
+        id: postBulkOperationsBody.id,
+      });
+
+      const getAsyncOperationBody = JSON.parse(getAsyncOperationResponse.body);
+
+      expect(getAsyncOperationResponse.statusCode).toEqual(200);
+      expect(getAsyncOperationBody.status).toEqual('SUCCEEDED');
+
+      let output;
+      try {
+        output = JSON.parse(getAsyncOperationBody.output);
+      } catch (error) {
+        throw new SyntaxError(`getAsyncOperationBody.output is not valid JSON: ${getAsyncOperationBody.output}`);
+      }
```

Comment:
Is it possible to have an `asyncOperation` with an invalid `output` if the record was written correctly and `SUCCEEDED`? Do/should we check on the write that the `output` is parsable JSON?

Reply:
I'm not sure if I understand - but by surrounding the JSON.parse in a try/catch and throwing an error if the JSON.parse throws, we are ensuring that the output is parsable JSON

Reply:
Yeah, I don't think it would need to be a separate check.